server/sonar-web/src/main/js/helpers/urls.ts

```diff
@@ -167,3 +167,23 @@ export function getMarkdownHelpUrl(): string {
 export function getCodeUrl(project: string, branch?: string, selected?: string) {
   return { pathname: '/code', query: { id: project, branch, selected } };
 }
+
+export function getOrganizationUrl(organization: string) {
+  return `/organizations/${organization}`;
+}
+
+export function getHomePageUrl(homepage: HomePage) {
+  switch (homepage.type) {
+    case HomePageType.Project:
+      return getProjectUrl(homepage.parameter!);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-1128&issues=AWB6ORyVUyqLuEgueoe5&open=AWB6ORyVUyqLuEgueoe5)

[ISSUE_KEY]: AWB6ORyVUyqLuEgueoe5